lib/FtpContext.js

```diff
@@ -204,18 +205,18 @@ module.exports = class FTPContext {
         // Only track control socket timeout during the lifecycle of a task associated with a handler.
         // That way we avoid timeouts on idle sockets, a behaviour that is not expected by most users.
         this.enableControlTimeout(true);
-        return new Promise((resolvePromise, rejectPromise) => {
+        return new Promise((resolve, reject) => {
             this._handler = handler;
             this._task = {
                 // When resolving or rejecting we also want the handler
                 // to no longer receive any responses or errors.
                 resolve: (...args) => {
                     this._stopTrackingTask();
-                    resolvePromise(...args);
+                    resolve(...args);
```

Comment:
It's `resolvePromise` to make it easier to discern from task.resolve… Don't feel strongly about it but I feel information for readers gets lost if renamed to `resolve`.